aead/Cargo.toml

```diff
@@ -7,8 +7,16 @@ license = "MIT OR Apache-2.0"
 description = "Traits for Authenticated Encryption with Associated Data (AEAD) algorithms"
 documentation = "https://docs.rs/aead"
 repository = "https://github.com/RustCrypto/traits"
-keywords = ["digest", "crypto", "encryption"]
+keywords = ["crypto", "encryption"]
 categories = ["cryptography", "no-std"]
 
 [dependencies]
 generic-array = { version = "0.12", default-features = false }
+heapless = { version = "0.5", optional = true }
```

Comment:
Alternatively we could define our own `heapless::Vec`-like type which wraps a `GenericArray<u8, N>` and does the minimum necessary to impl the `Buffer` trait, which would eliminate this dependency. But `heapless` seems relatively popular in the embedded space.